code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -145,6 +145,7 @@
 					/obj/item/weapon/gun/shotgun/double = 2,
 					/obj/item/weapon/gun/shotgun/pump/ksg = 2,
 					/obj/item/weapon/gun/shotgun/pump/bolt = 2,
+					/obj/item/weapon/gun/pistol/knife = 1
```

Comment:
This one is very much a no.

Reply:
Is it a damage issue, or a thematic issue?

Damage can be nerfed so a fully-kitted out one is less than machete, but more than bootknife-level.

Thematic-wise, I find it refreshingly funny and would like to see it actually appear in game in some capacity.

Reply:
Thematic. This is a knife that can take attachments and shoots. It doesn't fix our context.

Reply:
It can't shoot, it's just a knife. You CAN kit it out to have 50 damage, but that can be lowered to machete-tier.

Reply:
I mean, I SUPPOSE I could add in a magazine and ammo type for throwing knives, but as of NOW, it can't shoot :P

Reply:
It's a meme, in short. It's for the same reason we don't have clowns.

Reply:
> It's a meme, in short. It's for the same reason we don't have clowns.

You just gave me an idea.

We DO have clowns, actually. Clown survivors... oh wait, someone removed them. :(

Whatever, I'll remove it from the vendor and add it to one of the survivor loadouts next PR. Probably the ~~mallcop~~ security guard.

Reply:
We removed survivor clowns for a reason. Please, less memes.